xbmc/addons/include/xbmc_pvr_types.h

```diff
@@ -65,26 +65,73 @@ struct DemuxPacket;
 #define PRAGMA_PACK 1
 #endif
 
-#define PVR_ADDON_NAME_STRING_LENGTH         1024
-#define PVR_ADDON_URL_STRING_LENGTH          1024
-#define PVR_ADDON_DESC_STRING_LENGTH         1024
-#define PVR_ADDON_INPUT_FORMAT_STRING_LENGTH 32
-#define PVR_ADDON_EDL_LENGTH                 32
+#define PVR_ADDON_NAME_STRING_LENGTH          1024
+#define PVR_ADDON_URL_STRING_LENGTH           1024
+#define PVR_ADDON_DESC_STRING_LENGTH          1024
+#define PVR_ADDON_INPUT_FORMAT_STRING_LENGTH  32
+#define PVR_ADDON_EDL_LENGTH                  32
+#define PVR_ADDON_TIMERTYPE_ARRAY_SIZE        32
+#define PVR_ADDON_TIMERTYPE_VALUES_ARRAY_SIZE 512
+#define PVR_ADDON_TIMERTYPE_STRING_LENGTH     64
 
 /* using the default avformat's MAX_STREAMS value to be safe */
 #define PVR_STREAM_MAX_STREAMS 20
 
 /* current PVR API version */
-#define XBMC_PVR_API_VERSION "1.9.6"
+#define XBMC_PVR_API_VERSION "1.9.7"
 
 /* min. PVR API version */
-#define XBMC_PVR_MIN_API_VERSION "1.9.6"
+#define XBMC_PVR_MIN_API_VERSION "1.9.7"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
   /*!
+   * @brief numeric PVR timer type definitions (PVR_TIMER.iTimerType values)
+   */
+  const unsigned int PVR_TIMER_TYPE_NONE = 0; /*!< @brief "Null" value for a numeric timer type. */
+
+  /*!
+   * @brief PVR timer type attributes (PVR_TIMER_TYPE.iAttributes values)
+   */
+  const unsigned int PVR_TIMER_TYPE_ATTRIBUTE_NONE                    = 0x00000000;
+
+  const unsigned int PVR_TIMER_TYPE_IS_MANUAL                         = 0x00000001; /*!< @brief defines whether this is a type for manual (time-based) or epg-based timers */
```

Comment:
matter of taste, but I prefer `(1 << 0)`, `(1 << 1)`, `(1 << 2)`, etc for defining the values